src/webgl/material.js

```diff
@@ -178,7 +178,7 @@ p5.prototype.ambientMaterial = function(v1, v2, v3, a) {
   gl.useProgram(shaderProgram);
   shaderProgram.uMaterialColor = gl.getUniformLocation(
     shaderProgram, 'uMaterialColor' );
-  var colors = this._renderer._applyColorBlend(v1,v2,v3,a);
+  var colors = this._renderer._applyColorBlend.apply(this._renderer, arguments);
```

Comment:
@indefinit, for example `specularMaterial(250)`: `v1, v2, v3, a` is parsed in, the arguments in `_applyColorBlend` will become `[250, undefined, undefined, undefined]` instead of `[250]`. It will be translated to be red, instead of gray.
That's why we are seeing a red color in here http://p5js.org/examples/examples/3D_Multiple_Lights.php.